test/grammar_test.js

```diff
@@ -0,0 +1,117 @@
+        ['22-23-я линии В.О.', 'genitive', '22-23-й линий В.О.'],
+        ['22-23-я линии В.О.', 'prepositional', '22-23-й линиях В.О.'],
+        ['Австрийская площадь', 'accusative', 'Австрийскую площадь'],
+        ['Австрийская площадь', 'dative', 'Австрийской площади'],
+        ['Австрийская площадь', 'genitive', 'Австрийской площади'],
+        ['Австрийская площадь', 'prepositional', 'Австрийской площади'],
+        ['Лялина площадь', 'accusative', 'Лялину площадь'],
+        ['Лялина площадь', 'dative', 'Лялиной площади'],
+        ['Лялина площадь', 'genitive', 'Лялиной площади'],
+        ['Лялина площадь', 'prepositional', 'Лялиной площади'],
+        ['1-й Тверской-Ямской переулок', 'accusative', '1-й Тверской-Ямской переулок'],
+        ['1-й Тверской-Ямской переулок', 'dative', '1-му Тверскому-Ямскому переулку'],
+        ['1-й Тверской-Ямской переулок', 'genitive', '1-го Тверского-Ямского переулка'],
+        ['1-й Тверской-Ямской переулок', 'prepositional', '1-м Тверском-Ямском переулке'],
+        ['Большой Сампсониевский проспект', 'accusative', 'Большой Сампсониевский проспект'],
+        ['Большой Сампсониевский проспект', 'dative', 'Большому Сампсониевскому проспекту'],
+        ['Большой Сампсониевский проспект', 'genitive', 'Большого Сампсониевского проспекта'],
+        ['Большой Сампсониевский проспект', 'prepositional', 'Большом Сампсониевском проспекте'],
+        ['Нижний Лебяжий мост', 'accusative', 'Нижний Лебяжий мост'],
+        ['Нижний Лебяжий мост', 'dative', 'Нижнему Лебяжьему мосту'],
+        ['Нижний Лебяжий мост', 'genitive', 'Нижнего Лебяжьего моста'],
+        ['Нижний Лебяжий мост', 'prepositional', 'Нижнем Лебяжьем мосту'],
+        ['Старо-Калинкин мост', 'accusative', 'Старо-Калинкин мост'],
+        ['Старо-Калинкин мост', 'dative', 'Старо-Калинкину мосту'],
+        ['Старо-Калинкин мост', 'genitive', 'Старо-Калинкина моста'],
+        ['Старо-Калинкин мост', 'prepositional', 'Старо-Калинкином мосту'],
+        ['Тучков мост', 'accusative', 'Тучков мост'],
+        ['Тучков мост', 'dative', 'Тучкову мосту'],
+        ['Тучков мост', 'genitive', 'Тучкова моста'],
+        ['Тучков мост', 'prepositional', 'Тучковом мосту'],
+        ['Пыхов-Церковный проезд', 'accusative', 'Пыхов-Церковный проезд'],
+        ['Пыхов-Церковный проезд', 'dative', 'Пыхову-Церковному проезду'],
+        ['Пыхов-Церковный проезд', 'genitive', 'Пыхова-Церковного проезда'],
+        ['Пыхов-Церковный проезд', 'prepositional', 'Пыховом-Церковном проезде'],
+        ['Третье Транспортное кольцо', 'accusative', 'Третье Транспортное кольцо'],
+        ['Третье Транспортное кольцо', 'dative', 'Третьему Транспортному кольцу'],
+        ['Третье Транспортное кольцо', 'genitive', 'Третьего Транспортного кольца'],
+        ['Третье Транспортное кольцо', 'prepositional', 'Третьем Транспортном кольце']
+    ]
+    // TODO add your language grammar tests to call grammarize() and check result
```

Comment:
This is what the code below does, right?

Reply:
This is the place to add other language's test strings to check other language's grammar when it will be.